editor/src/components/canvas/canvas-strategies/strategies/flex-reorder-strategy.tsx

```diff
@@ -52,19 +54,19 @@ export function flexReorderStrategy(
       controlsToRender: [
         controlWithProps({
           control: DragOutlineControl,
-          props: dragTargetsElementPaths(selectedElements),
+          props: dragTargetsElementPaths(originalTargets),
           key: 'ghost-outline-control',
           show: 'visible-only-while-active',
         }),
         controlWithProps({
           control: ImmediateParentOutlines,
-          props: { targets: selectedElements },
+          props: { targets: originalTargets }, // question to Berci: should this be the original target? or the retargeted _ancestor_?
```

Comment:
originaltargets is good based on the research here https://github.com/concrete-utopia/utopia/pull/3458